discord/message.py

```diff
@@ -848,6 +854,12 @@ def __init__(
         except KeyError:
             self.interaction = None
 
+        self.poll: Poll | None
+        try:
+            self.poll = Poll.from_dict(data.get("poll"), message=self)
+        except:
```

Comment:
I think this is catching something that it should not be. I think there is an error in `Poll.from_dict` as none of the messages with polls in them show that they have polls when I fetch them. Will look into it further.

Reply:
Yes, the import for PartialEmoji was in the type-checking section. Also, two of the type-checking section imports are unused (`User`, and `Guild`).